src/go/guestagent/main.go

```diff
@@ -220,7 +187,7 @@ func main() {
 			// of the legacy network, requiring listeners only. In listenerOnlyMode, we create
```

Comment:
Line 185 of the comment still refers to privileged services.

It sounds like we never want `listenerOnlyMode` at all (since the comment says it should only be active in old network). Presumably we'll deal with that later in a separate PR?

Reply:
Yeah, I will make a follow-up PR to remove this one.